<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UISettings</class>
 <widget class="QDialog" name="UISettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTabWidget" name="twPages">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="wGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout1">
       <item row="17" column="0">
        <widget class="QCheckBox" name="cbClockSync">
         <property name="text">
          <string>Synchronize Saturn internal clock with set time</string>
         </property>
        </widget>
       </item>
       <item row="19" column="0">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>2</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="leBios"/>
       </item>
       <item row="1" column="1">
        <widget class="QToolButton" name="tbBios">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="lCdRom">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>CD-ROM</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QComboBox" name="cbCdRom"/>
       </item>
       <item row="5" column="0">
        <widget class="QLineEdit" name="leCdRom"/>
       </item>
       <item row="5" column="1">
        <widget class="QToolButton" name="tbCdRom">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QLabel" name="lSaveStates">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Save States</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLineEdit" name="leSaveStates"/>
       </item>
       <item row="7" column="1">
        <widget class="QToolButton" name="tbSaveStates">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QLabel" name="lTranslation">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Translation</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="12" column="0" colspan="2">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="13" column="0" colspan="2">
        <widget class="QCheckBox" name="cbEnableFrameSkipLimiter">
         <property name="text">
          <string>Enable Frame Skip/Limiter</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0" colspan="2">
        <widget class="QCheckBox" name="cbShowFPS">
         <property name="text">
          <string>Show FPS</string>
         </property>
        </widget>
       </item>
       <item row="15" column="0" colspan="2">
        <widget class="QCheckBox" name="cbAutostart">
         <property name="text">
          <string>Autostart</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="lBios">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Bios</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QComboBox" name="cbCdDrive">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="QComboBox" name="cbTranslation"/>
       </item>
       <item row="18" column="0">
        <widget class="QDateTimeEdit" name="dteBaseTime">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="wVideo">
      <attribute name="title">
       <string>Video</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout2">
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="lVideoCore">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Video Core</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QComboBox" name="cbVideoCore"/>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QLabel" name="lOSDCore">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>OSD Core</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QComboBox" name="cbOSDCore"/>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="QLabel" name="lWinResolution">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Window Resolution</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="3">
        <widget class="QComboBox" name="cbAspectRatio"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lWinWidth">
         <property name="text">
          <string>Width</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="leWinWidth"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="lWinHeight">
         <property name="text">
          <string>Height</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="leWinHeight"/>
       </item>
       <item row="16" column="0" colspan="3">
        <widget class="QLabel" name="lVideoFormat">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Video Format</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="17" column="0" colspan="3">
        <widget class="QComboBox" name="cbVideoFormat"/>
       </item>
       <item row="18" column="0" colspan="3">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>2</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="lFullScreenResolution">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Fullscreen Resolution</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QCheckBox" name="cbFullscreen">
         <property name="text">
          <string>Start in Fullscreen</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="3">
        <widget class="QComboBox" name="cbFullscreenResolution"/>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="cbBilinear">
         <property name="text">
          <string>Bilinear Filtering (Restart required)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="wSound">
      <attribute name="title">
       <string>Sound</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QLabel" name="lSoundCore">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Sound Core</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbSoundCore"/>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>2</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="wCartMemory">
      <attribute name="title">
       <string>Cart/Memory</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout3">
       <item row="5" column="3">
        <widget class="QToolButton" name="tbCartridge">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="7" column="3">
        <widget class="QToolButton" name="tbMemory">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="3">
        <widget class="QLineEdit" name="leMemory"/>
       </item>
       <item row="9" column="3">
        <widget class="QToolButton" name="tbMpegROM">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>2</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="QLineEdit" name="leCartridge">
         <property name="inputMask">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="4">
        <widget class="QLabel" name="lMemory">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Memory</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="4">
        <widget class="QLabel" name="lMpegROM">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Mpeg ROM</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="4">
        <widget class="QLabel" name="lCartridge">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Cartridge</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="3">
        <widget class="QLineEdit" name="leMpegROM"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lCartridgeModemIP">
         <property name="text">
          <string>Default Dial IP</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="2">
        <widget class="QLineEdit" name="leCartridgeModemIP">
         <property name="toolTip">
          <string/>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="maxLength">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QComboBox" name="cbCartridge"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lCartridgeModemPort">
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2" colspan="2">
        <widget class="QLineEdit" name="leCartridgeModemPort">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="maxLength">
          <number>5</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="wInput">
      <attribute name="title">
       <string>Input</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout4">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="lInput">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Input</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QComboBox" name="cbInput"/>
       </item>
       <item row="2" column="0">
        <widget class="UIPortManager" name="pmPort1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Port 1</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="UIPortManager" name="pmPort2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Port 2</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lGunMouseSensitivity">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Gun/Mouse Sensitivity</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QSlider" name="sGunMouseSensitivity">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>400</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="wAdvanced">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QLabel" name="lRegion">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Region</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbRegion"/>
       </item>
       <item>
        <widget class="QLabel" name="lSH2Interpreter">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>SH2 Interpreter</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbSH2Interpreter"/>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>2</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="wView">
      <attribute name="title">
       <string>View</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="6" column="0">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>2</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="lHideMenubar">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Hide Menubar</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="rbMenubarNever">
         <property name="text">
          <string>Never</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">bgShowMenubar</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="rbMenubarFullscreen">
         <property name="text">
          <string>On fullscreen</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">bgShowMenubar</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QRadioButton" name="rbMenubarAlways">
         <property name="text">
          <string>Always</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">bgShowMenubar</string>
         </attribute>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QLabel" name="lHideToolbar">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Hide Toolbar</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="rbToolbarNever">
         <property name="text">
          <string>Never</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">bgShowToolbar</string>
         </attribute>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QRadioButton" name="rbToolbarFullscreen">
         <property name="text">
          <string>On fullscreen</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">bgShowToolbar</string>
         </attribute>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QRadioButton" name="rbToolbarAlways">
         <property name="text">
          <string>Always</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">bgShowToolbar</string>
         </attribute>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Show Log Window</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QRadioButton" name="rbLogWindowMessage">
         <property name="text">
          <string>On message</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">bgShowLogWindow</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QRadioButton" name="rbMenubarFullscreenHover">
         <property name="text">
          <string>Display on Hover</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">bgShowMenubar</string>
         </attribute>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QRadioButton" name="rbLogWindowNever">
         <property name="text">
          <string>Never</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">bgShowLogWindow</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Shortcuts</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="UIShortcutManager" name="twShortcuts">
         <property name="tabKeyNavigation">
          <bool>false</bool>
         </property>
         <property name="dragDropOverwriteMode">
          <bool>false</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Description</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Shortcut</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dbbButtons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UIPortManager</class>
   <extends>QGroupBox</extends>
   <header>ui/UIPortManager.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UIShortcutManager</class>
   <extends>QTableWidget</extends>
   <header>UIShortcutManager.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>dbbButtons</sender>
   <signal>accepted()</signal>
   <receiver>UISettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dbbButtons</sender>
   <signal>rejected()</signal>
   <receiver>UISettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="bgShowLogWindow"/>
  <buttongroup name="bgShowToolbar"/>
  <buttongroup name="bgShowMenubar"/>
 </buttongroups>
</ui>
